projects/plugins/jetpack/modules/seo-tools/class-jetpack-seo-posts.php

```diff
@@ -10,19 +10,21 @@
  */
 class Jetpack_SEO_Posts {
 	/**
-	 * Key of the post meta value that will be used to store post custom description.
+	 * Key of the post meta values that will be used to store post custom data.
 	 */
 	const DESCRIPTION_META_KEY = 'advanced_seo_description';
+	const HTML_TITLE_META_KEY  = 'jetpack_seo_html_title';
 
 	/**
 	 * Build meta description for post SEO.
 	 *
-	 * @param WP_Post $post Source of data for custom description.
+	 * @param WP_Post|null $post Source of data for custom description.
 	 *
 	 * @return string Post description or empty string.
 	 */
 	public static function get_post_description( $post ) {
```

Comment:
With this coalescing `$post` to a WP_Post object if omitted, it should probably be `( $post = null )` to set a default if it's ever desired to be used without passing in a post id.

Reply:
Makes sense to me 👍 e1007f6